### PR TITLE
feat: Cleanup Sample App

### PR DESCRIPTION
**Pull request by pat-deque**
Resolves: [#179](https://app.zenhub.com/workspaces/mobile---android-618c52b298700300142de56d/issues/gh/dequelabs/mobile-general/179)

### CODE REVIEW

**Review by devchan4188 — APPROVED** [review]
LGTM!

**Comment by devchan4188** [comment]
If you want you can do this [issue](https://app.zenhub.com/workspaces/mobile-5ccf5c056202ac26f233076a/issues/gh/dequelabs/mobile-general/178) in this PR or ignore the tests in this PR and make a separate PR for the above issue. Your call.

**Comment by pat-deque** [comment]
> If you want you can do this [issue](https://app.zenhub.com/workspaces/mobile-5ccf5c056202ac26f233076a/issues/gh/dequelabs/mobile-general/178) in this PR or ignore the tests in this PR and make a separate PR for the above issue. Your call.

I really wanted to get back to this, but other items need my attention. We'll come back to this when we have bandwidth

File: app/src/androidTest/java/com/deque/mobile/axedevtoolssampleapp/apiexamples/InstrumentationRegistryExampleTest.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+class InstrumentationRegistryExampleTest {
+
+    @Rule
+    @JvmField
+    val rule: ActivityScenarioRule<MainActivity> = ActivityScenarioRule(MainActivity::class.java)
+
+    val axe = AxeDevTools()
+
+    init {
+        axe.loginWithApiKey("6a6cedfa-499b-48a0-bc2b-c7a50125d696", AxeDevToolsClient.DB_QA_URL)
```
Comment (devchan4188) [inline app/src/androidTest/java/com/deque/mobile/axedevtoolssampleapp/apiexamples/InstrumentationRegistryExampleTest.kt:33]: Can you please get rid of this?

**Review by devchan4188 — APPROVED** [review]
LGTM!

**Pull request merged**